data/human/outfits.txt

```diff
@@ -105,24 +105,122 @@ outfit "S-270 Regenerator"
 	thumbnail "outfit/small regenerator"
 	"mass" 12
 	"outfit space" -12
-	"shield generation" .6
-	"shield energy" .6
+	"shield generation" 1.32
+	"shield energy" 1.32
```

Comment:
I presume you meant to give this and the S-970 a shield delay? This is more than the 50% increase you mentioned.